docs/scalardb-server.md

```diff
@@ -89,17 +96,17 @@ $ docker pull ghcr.io/scalar-labs/scalardb-server:<version>
 
 And then, you can start Scalar DB Server with the following command:
 ```shell
-$ docker run -v <your local property file path>:/scalardb/server/database.properties.tmpl -d -p 60051:60051 -p 8080:8080 ghcr.io/scalar-labs/scalardb-server:<version>
+$ docker run -v <your local property file path>:/scalardb/server/database.properties -d -p 60051:60051 -p 8080:8080 ghcr.io/scalar-labs/scalardb-server:<version>
```

Comment:
Now, we don't need to use the `.tmpl` file for the `dockerize` command.
In other words, we can mount the `database.properties` file as is on the container.